### PR TITLE
Java EE8 Support

### PR DESCRIPTION
**Pull request by leochr**
- Add javaee8 and webProfile8
- Update list of test images and sync repository (tools) to include EE8

### CODE REVIEW

File: tools/syncRepository.sh [hunk 1]
```diff
@@ -67,7 +67,9 @@ then
     pullAndSync $target common
     pullAndSync $target webProfile6
```
Comment (leochr) [inline tools/syncRepository.sh]: Should webProfile6 removed from sync? Not entirely sure.

File: test/images.txt [hunk 2]
```diff
@@ -2,5 +2,7 @@ websphere-liberty:kernel       ../ga/developer/kernel
 websphere-liberty:microProfile  ../ga/developer/microProfile
 websphere-liberty:webProfile6  ../ga/developer/webProfile6
```
Comment (arthurdm) [inline test/images.txt]: maybe remove this too?

File: tools/syncRepository.sh (hunk 1 above)
Comment (arthurdm) [reply tools/syncRepository.sh]: I think so.

**Comment by leochr** [comment]
@arthurdm please review the new commits - including the changes to centos and rhel (updated the Liberty version to 18.0.0.2 in Dockerfiles).

**Comment by arthurdm** [comment]
thanks @leochr - looks good.

File: ga/developer/javaee8/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+FROM websphere-liberty:kernel
+
+ARG REPOSITORIES_PROPERTIES=""
+
+RUN if [ ! -z $REPOSITORIES_PROPERTIES ]; then mkdir /opt/ibm/wlp/etc/ \
+  && echo $REPOSITORIES_PROPERTIES > /opt/ibm/wlp/etc/repositories.properties; fi \
+  && installUtility install --acceptLicense \
+    appSecurity-2.0 collectiveMember-1.0 ldapRegistry-3.0 \
+    localConnector-1.0 monitor-1.0 requestTiming-1.0 restConnector-2.0 \
+    sessionDatabase-1.0 ssl-1.0 transportSecurity-1.0 webCache-1.0 webProfile-8.0 \
```
Comment (aguibert) [inline ga/developer/javaee8/Dockerfile]: we should also add `sessionCache-1.0` here, which was a new feature in 18.0.0.1 similar to `sessionDatabase-1.0`

File: ga/developer/webProfile8/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+FROM websphere-liberty:kernel
+
+ARG REPOSITORIES_PROPERTIES=""
+
+RUN if [ ! -z $REPOSITORIES_PROPERTIES ]; then mkdir /opt/ibm/wlp/etc/ \
+  && echo $REPOSITORIES_PROPERTIES > /opt/ibm/wlp/etc/repositories.properties; fi \
+  && installUtility install --acceptLicense \
+    appSecurity-2.0 collectiveMember-1.0 ldapRegistry-3.0 \
+    localConnector-1.0 monitor-1.0 requestTiming-1.0 restConnector-2.0 \
+    sessionDatabase-1.0 ssl-1.0 transportSecurity-1.0 webCache-1.0 webProfile-8.0 \
```
Comment (aguibert) [inline ga/developer/webProfile8/Dockerfile]: add `sessionCache-1.0` here too

**Comment by leochr** [comment]
@aguibert added sessionCache-1.0. thanks.

**Pull request merged**